src/App.js

```diff
@@ -1,5 +1,32 @@
+import CarRace from './Model/CarRace.js';
+import InputView from './View/InputView.js';
+import OutputView from './View/OutputView.js';
+
 class App {
-  async run() {}
+  #input_view = new InputView();
+
+  #output_view = new OutputView();
```

Comment:
갠적으로 상수가 아닌 변수명에 언더바(_)를 넣는 건 상수랑 헷갈릴 여지가 있을 것 같아서,
변수명을 카멜케이스로 작성하는 건 어떤가 싶습니당.
```suggestion
  #inputView = new InputView();

  #outputView = new OutputView();
```

Reply:
엇 변수명을 스네이크 케이스로 지었네요🥲

꼼꼼한 리뷰 감사합니다!